Patch for Ditz to add bug/feature tag display to issue names.

diff --git a/lib/html.rb b/lib/html.rb
--- a/lib/html.rb
+++ b/lib/html.rb
@@ -71,7 +71,7 @@ class ErbHtml
   end
 
   def issue_link_for i, opts={}
-    link = link_to i, "#{i.title}"
+    link = link_to i, "#{i.tag}"
     link = "<span class=\"inline-issue-link\">" + link + "</span>" if opts[:inline]
     link = link + " " + issue_status_img_for(i, :class => "inline-status-image") if opts[:status_image]
     link
diff --git a/lib/model-objects.rb b/lib/model-objects.rb
--- a/lib/model-objects.rb
+++ b/lib/model-objects.rb
@@ -228,6 +228,13 @@ class Issue < ModelObject
     change_status :paused, who, comment
   end
 
+  def tag
+    s = self.title
+    s = s + " (bug)" if self.bug?
+    s = s + " (feature)" if self.feature?
+    return s
+  end
+
   def close disp, who, comment
     raise Error, "unknown disposition #{disp}" unless DISPOSITIONS.member? disp
     log "closed with disposition #{disp}", who, comment
diff --git a/lib/operator.rb b/lib/operator.rb
--- a/lib/operator.rb
+++ b/lib/operator.rb
@@ -304,7 +304,7 @@ EOS
     return if issues.empty?
     name_len = issues.max_of { |i| i.name.length }
     issues.map do |i|
-      s = sprintf "%s %#{name_len}s: %s", i.status_widget, i.name, i.title
+      s = sprintf "%s %#{name_len}s: %s", i.status_widget, i.name, i.tag
       s += " [#{i.release}]" if opts[:show_release] && i.release
       s + "\n"
     end.join
